src/Promact.TestCaseManagement.DomainModel/Enums/Condition.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Promact.TestCaseManagement.DomainModel.Enums
+{
+    public enum Condition
+    {
+        PreCondition,
```

Comment:
I think there is still confusion with Pre-Condition and Post-Condition.

Pre-Condition can have 
1. Previous test case as a condition
2. Simple string
3. Mixture of both

Post-Condition will have only string value nothing else. according to that there is no similarity between Pre-Condition and Post-Condition.